.github/workflows/gluten.yml

```
@@ -55,20 +49,45 @@ jobs:
         id: gluten-cache
         uses: actions/cache@v4
         with:
-          path: gluten/package/target/
+          path: incubator-gluten/package/target/
           key: gluten_package_${{ steps.date.outputs.date }}
-      - name: Build gluten project
+      - name: Get Ccache
+        id: gluten-ccache-cache
+        uses: actions/cache@v4
+        with:
+          path: incubator-gluten/.ccache
+          key: ccache-centos7-release-default-${{github.sha}}
+          restore-keys: |
+            ccache-centos7-release-default
+      - name: Build Gluten velox third party
         run: |
           if [[ "${{ steps.gluten-cache.outputs.cache-hit }}" != 'true' ]]; then
-            git clone https://github.com/oap-project/gluten.git
-            cd gluten
-            ./dev/buildbundle-veloxbe.sh
+            git clone https://github.com/apache/incubator-gluten.git
```

use `--depth=1` to skip retrieving git history commits to speed up the clone
```suggestion
            git clone https://github.com/apache/incubator-gluten.git --depth=1
```